src/bin/status_icon.rs

```diff
@@ -0,0 +1,89 @@
+//! # TreeView Sample
+//!
+//! This sample demonstrates how to create a StatusIcon.
+
+extern crate gdk_pixbuf;
```

Comment:
IMHO this file needed `#![allow(deprecated)]`, see https://travis-ci.org/gtk-rs/examples/jobs/432129991#L721